css/index.css

```diff
@@ -111,36 +136,281 @@ p {
+    padding-bottom: 20px;  
+}
+
+.medium2-content {
+    display: block;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    margin-bottom: 20px;
+    border-bottom: 1px dashed black;
+    margin-top: 20px;
+}
+
+.medium2-content .text-container {
+    width: 48%;
+    padding: 0 1%;
+    padding-bottom: 20px;  
+}
+
+/*.title-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    margin-bottom: 20px;
+    border-bottom: 1px dashed black;
+    margin-top: 20px;
+}
+.title-content .text-container {
+    width: 48%;
+    padding: 0 1%;
+    padding-bottom: 20px; 
+} */
+
+.title-content {
+    display: block;
+    margin: 0 2% 20px;
+    justify-content: space-around;
+    margin-bottom: 20px;
+    border-bottom: 1px dashed black;
+    margin-top: 20px;
+    padding: 5px;
+}
+
+.together-content {
+    display: block;
+    margin: 0 2% 20px;
+    justify-content: space-around;
+    margin-bottom: 20px;
+    margin-top: 20px;
+}
+
+.medium-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    line-height: 1.3;
+}
+
+.medium-content .medium-header-img {
+    margin: 20px 0;
+}
+
+.medium-content h1 {
+    font-size: 26px;
+    margin: 10px 0;
+}
+
+.medium-content .medium-text {
+ border-bottom: 1px solid black;
+ padding-bottom: 40px;
+ margin-bottom: 20px
+}
+
+.medium-content .card {
+    padding: 20px;
+    width: 350px;
+    margin: 20px 0;
+}
+
+.medium-content .card button {
+    padding: 10px 40px;
+    margin-top: 20px;
+    cursor: pointer;
+    background: white;
+    border-radius: 10px;
+    border: 1px solid black;
+    
+}
+
+.medium-content .card button:hover {
+    background: black;
+    color: white;
+} 
+
+.medium-content .card {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    line-height: 1.3;
+}
+
+.medium2-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    line-height: 1.3;
+}
+
+.medium2-content .medium2-header-img {
+    margin: 20px 0;
+}
+
+.medium2-content h1 {
+    font-size: 26px;
+    margin: 10px 0;
+}
+
+.medium2-content .medium2-text {
+ border-bottom: 1px solid black;
+ padding-bottom: 40px;
+ margin-bottom: 20px
+}
+
+.medium2-content .card {
+    padding: 20px;
+    width: 350px;
+    margin: 20px 0;
+}
+
+.medium2-content .card button {
+    padding: 10px 40px;
+    margin-top: 20px;
+    cursor: pointer;
+    background: white;
+    border-radius: 10px;
+    border: 1px solid black;
+    
+}
+
+.medium2-content .card button:hover {
+    background: black;
+    color: white;
+} 
+
+.medium2-content .card {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    line-height: 1.3;
+}
+
+.south {
+    width: 45%;
+    background: black;
+    position: absolute;
+    margin-left: 550px;
+
+}
+.south nav {
+    width: 60%;
+    display: flex;
+    justify-content: space-between;
+    padding: 20px 2%;
+    font-size: 14px;
```

Comment:
CSS looks good! So that your CSS doesn't become too cluttered and too long. It might be a good idea to separate it into different files!